### ISSUE TITLE
gherkin-testcafe file is causing anti-virus to detect malware

### ISSUE THREAD

**Issue by anthonyjsargeant** [kiwigrid/gherkin-testcafe#55]
I've recently added gherkin-testcafe to a project I'm working on. When running `npm install` on the project, it pulls in gherkin-testcafe as a dependency, but my anti-virus is now detecting malware as in the screenshot below:

![Screenshot 2019-11-13 at 11 33 41](https://user-images.githubusercontent.com/10552344/68763532-978c0200-0610-11ea-8d68-ea363e930e1b.png)

The file in question is:
```
node_modules/gherkin-testcafe/node_modules/gherkin/executables/gherkin-windows-386.exe
```

This has been detected by other members of the team. Whilst this could be a false-positive, this is something that you may want to take a look at.

**Comment by Lukas-Kullmann** [comment]
Thanks for the notice, @anthonyjsargeant.

I just released v2.4.2 that updates gherkin to v8.1.1. I hope that fixes the issue.
If you still have this problem with the new version, please file a bug at https://github.com/cucumber/cucumber/

**Comment by anthonyjsargeant** [comment]
Thanks @Lukas-Kullmann for the prompt response!
